cmd/kubebuilder/v1/api.go

```diff
@@ -50,11 +50,24 @@ func (o *apiOptions) RunAddAPI() {
 		fmt.Println("Create Resource under pkg/apis [y/n]?")
 		o.doResource = util.Yesno(reader)
 	}
+
 	if !o.controllerFlag.Changed {
 		fmt.Println("Create Controller under pkg/controller [y/n]?")
 		o.doController = util.Yesno(reader)
 	}
 
+	if o.doResource || o.doController {
```

Comment:
This is redundant because one of the two needs to be `true`, so lets remove that.

Reply:
Just out of curiosity, where is the check that it must be at least one of them?